<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myExchangeFormStock</class>
 <widget class="QWidget" name="myExchangeFormStock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>661</width>
     <height>97</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0">
    <property name="spacing">
     <number>10</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayoutAsset" rowstretch="1,1,1" columnstretch="1,2,1,2">
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <item row="2" column="2">
       <widget class="QLabel" name="labelPrice">
        <property name="text">
         <string>单价：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>spinBoxPrice</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelAmount">
        <property name="text">
         <string>数量：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QDoubleSpinBox" name="spinBoxPrice">
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelName">
        <property name="text">
         <string>名称：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>nameLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelCode">
        <property name="text">
         <string>代号：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>codeLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="codeLineEdit"/>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="nameLineEdit"/>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="hlMarket">
        <item>
         <widget class="QRadioButton" name="radioSH">
          <property name="text">
           <string>上海</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioSZ">
          <property name="text">
           <string>深圳</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioOther">
          <property name="text">
           <string>其它</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="3">
       <layout class="QHBoxLayout" name="hlBuySell">
        <item>
         <widget class="QRadioButton" name="radioBuy">
          <property name="text">
           <string>买入</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioSell">
          <property name="text">
           <string>卖出</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <widget class="myLineEdit" name="amountLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="horizontalSpacing">
       <number>1</number>
      </property>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="moneySpinBox">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="minimum">
         <double>-1000000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="moneySpinBoxRemain">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="minimum">
         <double>-1000000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelMoneyRemain">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>剩余资金：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>moneySpinBoxRemain</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>元</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>元</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelMoney">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>所需资金：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>moneySpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelMoneyOrigin">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>原始资金：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>元</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="moneySpinBoxTotal">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="minimum">
         <double>-1000000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>6</y>
     <width>551</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vlMoneyAccount" stretch="0,0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="10">
      <item>
       <widget class="QLabel" name="labelaccountMoney">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>交易帐户</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,0">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QComboBox" name="moneyAccount"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,3,1">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="feeRateSpinBox">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelFee">
          <property name="text">
           <string>佣金：</string>
          </property>
          <property name="buddy">
           <cstring>feeRateSpinBox</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="labelFeeUnit">
          <property name="text">
           <string>‰</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myLineEdit</class>
   <extends>QLineEdit</extends>
   <header>customUI/myLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>moneyAccount</tabstop>
  <tabstop>radioSH</tabstop>
  <tabstop>radioSZ</tabstop>
  <tabstop>radioOther</tabstop>
  <tabstop>radioBuy</tabstop>
  <tabstop>radioSell</tabstop>
  <tabstop>codeLineEdit</tabstop>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>spinBoxPrice</tabstop>
  <tabstop>moneySpinBoxTotal</tabstop>
  <tabstop>moneySpinBox</tabstop>
  <tabstop>moneySpinBoxRemain</tabstop>
  <tabstop>feeRateSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
